mapstore/proxy.properties

```diff
@@ -17,8 +17,8 @@ defaultStreamByteSize=1024
 # White lists, if defined, limit the requests that are accepted and handled by the proxy. Any request that is not
 # matching a defined whitelist will be rejected.
 
-# hostname whitelist limit the accepted requesting hosts (where the client application is hosted), to avoid the proxy being used by anyone
-# you should set the list to accept only the host(s) where mapstore is deployed
+# hostname whitelist limit the accepted target hosts (where the proxy is permitted to forward to)
```

Comment:
hostnameWhitelist defines the accepted target hosts for the internal proxy.
The list should be set to the hosts whose CORS headers do not allow direct access